crates/chisel/src/dispatcher.rs

```diff
@@ -301,7 +301,7 @@ impl ChiselDispatcher {
                         self.session.id.as_ref().unwrap()
                     )))
                 } else {
-                    DispatchResult::CommandFailed(Self::make_error("Failed to load session!"))
+                    DispatchResult::CommandFailed(Self::make_error("Failed to load session! The session ID may not exist or might be corrupted."))
```

Comment:
since session id is something chisel internal, I dont think we should include this in the error message